### ISSUE TITLE
Wrong language in Page Options aria-label on Spanish pages

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/cgov-digital-platform#2685]
### Issue description
The value of the aria-label attributes in the Page Options box on Spanish pages are in English.

> **ESTIMATE** 2

### Steps to reproduce the issue

1.  Go to any Spanish page which includes Page Options.  e.g. https://www.cancer.gov/espanol/investigacion/areas/examenes-de-deteccion
2. View page source
3. Locate the page options section

### What's the expected result?
- The values in aria-label attributes should be in Spanish.

### What's the actual result?
- The values in aria-label attributes are in English.

### Additional details / screenshot
```html
  <div id="PageOptionsControl1" class="page-options no-resize">
      <ul>
                  <li class="page-options--resize">
            <a title="Resize font" aria-label="Resize font" href="#"></a>
          </li>
                          <li class="page-options--print">
            <a title="Print" aria-label="Print" href="#"></a>
          </li>
                          <li class="page-options--email">
            <a title="Email" aria-label="Email" href="#"></a>
          </li>
                          <li class="social-share social-share--facebook">
            <a title="Facebook" aria-label="Facebook" href="#"></a>
          </li>
                          <li class="social-share social-share--twitter">
            <a title="Twitter" aria-label="Twitter" href="#"></a>
          </li>
                          <li class="social-share social-share--pinterest">
            <a title="Pinterest" aria-label="Pinterest" href="#"></a>
          </li>
              </ul>
  </div>
  <!-- END PAGE OPTIONS -->```

### Related Tickets
* Issue #2684

**Comment by sarinapadilla** [comment]
This has been fixed with MVP1 of redesign!
